internal/tools/clickhouse_create_logs_table.sql

```diff
@@ -20,11 +20,6 @@ CREATE TABLE base.logs (
     INDEX topic1_idx topic_1 TYPE bloom_filter GRANULARITY 1,
     INDEX topic2_idx topic_2 TYPE bloom_filter GRANULARITY 1,
     INDEX topic3_idx topic_3 TYPE bloom_filter GRANULARITY 1,
-) ENGINE = SharedReplacingMergeTree(
-    '/clickhouse/tables/{uuid}/{shard}',
-    '{replica}',
-    insert_timestamp,
-    is_deleted
-)
-ORDER BY (block_number, transaction_hash, log_index) SETTINGS index_granularity = 8192
+) ENGINE = SharedReplacingMergeTree(insert_timestamp, is_deleted)
```

Comment:
what is the implication of this change?

Reply:
Removed default index_granularity and `allow_experimental_replacing_merge_with_cleanup` will allow deleting rows marked as deleted